### ISSUE TITLE
How do I attach to a running hhvm server in docker?

### ISSUE THREAD

**Issue by beatscode** [facebook/nuclide#930]
### Issue and Steps to Reproduce
I have a set of containers that make up my application. I have exposed port 9001 to the host

Here is what the xdebug portion of my ini file looks like:
```
;xdebug.dump.SERVER = *

; Xdebug options
xdebug.enable=1
xdebug.remote_enable=1
xdebug.idekey="xyz"
xdebug.remote_host="localhost"
xdebug.remote_handler=dbgp
xdebug.remote_port=9001
xdebug.remote_autostart=1
xdebug.remote_connect_back=1

xdebug.profiler_enable=0
xdebug.profiler_append=0

xdebug.collect_includes=0
xdebug.profiler_output_name=cachegrind.out.%p.%s
xdebug.profiler_output_dir=/var/www/app/storage/cachegrind

xdebug.auto_trace=0
xdebug.trace_output_dir=/var/www/app/storage/cachegrind
xdebug.trace_output_name="trace.%t"
xdebug.trace_format=1

;xhprof.output_dir=/var/www/app/storage/cachegrind
```

I’m following the directions from the following: https://nuclide.io/docs/features/task-runner/.
The play button is disabled and I can’t enter anything into “Attach to WebServer” section. The docs mention "If you select Attach to WebServer, the text box will fill automatically with the server to which you are connected”. Am i missing a configuration option here?

![b2338c45-d44f-4cd1-9f28-1f965082feb0](https://cloud.githubusercontent.com/assets/1777648/21185430/9cd3ae12-c1de-11e6-81cc-66b4c9624ef1.png)
![43666e3a-4340-46e7-aa35-c896b600f996](https://cloud.githubusercontent.com/assets/1777648/21185442/a4883cd6-c1de-11e6-94fe-bb0654ca31f7.png)

**Debug Settings**
![1bb15a74-833a-4c38-9971-203506360f78](https://cloud.githubusercontent.com/assets/1777648/21185447/aaaebbe4-c1de-11e6-87d9-69ac11865941.png)

### Expected Behavior
I expect to be able to select hhvm and start debug session

### Actual Behavior
Attach to Webserver is disabled

### Versions
* Atom: 1.12.7
* Nuclide: 0.191.0
* Client OS: Mac OSX Sierra 10.12.1

**Comment by johnislarry** [comment]
Hmm this changed somewhat recently, the play button being active is dependent on setting your `current working directory` (right click in the file tree on your php project root directory and choose "set current working directory").  Your project should also have an .hhconfig at the top I believe.  If this doesn't work, then you can go through the debugger selection UI, (cmd + shift + A) and select PHP and 'attach'

**Comment by johnislarry** [comment]
Closing this out, feel free to re-open if my suggestions don't work for you.

**Comment by beatscode** [comment]
Tried this again. I'm starting to believe it's not possible to connect to a docker container process. I'm using docker for mac by the way. In the image below all of the processes shown are of my local version of hhvm not the container's version. 
Is it possible to connect to the container remotely? I tried clicking "localhost" link in the attach debugger modal, but this only shows "localhost". Can't find any settings in nuclide that allows for me to add another ip.
<img width="500" alt="8b382ad5-f280-4a7f-859c-dd84f0c65f08" src="https://user-images.githubusercontent.com/1777648/27241674-c802e238-52a7-11e7-9e7d-8592ded37224.png">
